packages/kit/src/runtime/app/stores.js

```diff
@@ -1,7 +1,6 @@
 import { getContext } from 'svelte';
 
-// const ssr = (import.meta as any).env.SSR;
-const ssr = typeof window === 'undefined'; // TODO why doesn't previous line work in build?
+const ssr = import.meta.env.SSR;
```

Comment:
This broke some test mocks on our end. What's the best way to get `import.meta.env` to be defined in a non-vite context such as Jest or another test runner?

Reply:
there's an issue about this. it's been one of the things to figure out before 1.0: https://github.com/sveltejs/kit/issues/1485

Reply:
@wallw-bits @benmccann 

I noticed this breaking tests for me as well - I think something like 

```js
import { browser } from './env.js'

const ssr = !browser
```

would do the trick, as then you can just mock `$app/env.js` with

```
jest.mock('$app/env.js', () => ({
  amp: false,
  browser: true, // or false for testing ssr
  dev: true,
  mode: 'test'
}))
```

Reply:
just tried it - making that change locally did work

![image](https://user-images.githubusercontent.com/474827/131888770-e155028e-5108-4159-aa87-e452742eddb2.png)

I think it also makes the code more DRY - import.meta.env.SSR is already being dealt with in env.

I'll make a PR.

Reply:
PR made: https://github.com/sveltejs/kit/pull/2353